ios/browser/api/brave_wallet/brave_wallet_api.mm

```diff
@@ -26,8 +27,8 @@
     @"ethereum_provider.js";
 BraveWalletProviderScriptKey const BraveWalletProviderScriptKeySolana =
     @"solana_provider.js";
-BraveWalletProviderScriptKey const BraveWalletProviderScriptKeySolanaInternal =
-    @"solana_provider_internal.js";
+BraveWalletProviderScriptKey const BraveWalletProviderScriptKeySolanaWeb3JS =
```

Comment:
The `ios/browser/api/brave_wallet/brave_wallet_api.h` header file will need updated for this name change too:
https://github.com/brave/brave-core/blob/68e162730a2fda99cf61ff107bf5ae44a5708117/ios/browser/api/brave_wallet/brave_wallet_api.h#L25-L26

Reply:
fixed in a90423f98bf26d062849f2ad624b9ba2cacc7863

Reply:
Not fixed, ICYMI: https://github.com/brave/brave-core/commit/a90423f98bf26d062849f2ad624b9ba2cacc7863#r85274366

Reply:
24fcb4ffd34bfb42a6c45f4bc6f6b78e810789cc
should fix it now